apps/hermes/server/src/api/rest/v2/timestamp_price_updates.rs

```diff
@@ -67,6 +67,9 @@ pub struct TimestampPriceUpdatesQueryParams {
     /// If true, include the parsed price update in the `parsed` field of each returned feed. Default is `true`.
     #[serde(default = "default_true")]
     parsed: bool,
+
+    /// If true, invalid price IDs in the `ids` parameter are ignored. Only applicable to the v2 APIs. Default is `false`.    #[serde(default)]
```

Comment:
I think `#[serde(default)]` is commented by mistake

Reply:
oof good catch!